app/src/main/res/menu/bottom_menu.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+        <item
+            android:id="@+id/action_countdown"
+            android:enabled="true"
+            android:title="Countdown"
+            android:icon="@drawable/ic_shopping_cart_black_24dp"
+            app:showAsAction="ifRoom"
+            />
+
+        <item
+            android:id="@+id/action_paknsave"
+            android:enabled="true"
+            android:title="Pak'n'Save"
+            android:icon="@drawable/ic_school_black_24dp"
+            app:showAsAction="ifRoom"
+            />
+        <item
+            android:id="@+id/action_newworld"
+            android:enabled="true"
+            android:title="New World"
+            android:icon="@drawable/ic_restaurant_black_24dp"
+            app:showAsAction="ifRoom"
+            />
+        <!--
```

Comment:
Is this commented code needed? (Lines 27-35) Or is it for future work?

Reply:
That menu button is for potential back button depending on feedback of UX usability, will keep there for now incase we wish to re-implement it